dashboard/test/ui/features/foundations/header.feature

```diff
@@ -2,22 +2,24 @@
 @single_session
 Feature: Header navigation bar
 
-Scenario: Signed out user in English should see 6 header links
```

Comment:
Would you mind merging in staging please so it's clearer which changes are specific to this PR?

Reply:
Oh sorry I meant merging in the signed-out changes (I figured those were already in staging, my bad)

Reply:
Ok should be good now, sorry I merged incorrectly the first time